modules/network-firewall/README.md

```diff
@@ -11,11 +11,13 @@ The default firewall rules on GCP block inbound traffic and allow outbound traff
 connection is allowed between a source and a target or a target and a destination, all subsequent traffic in either
 direction will be allowed as long as the connection is active.
 
-This module adds rules for 3 [network `tags`](https://cloud.google.com/vpc/docs/add-remove-network-tags) that can be
+This module adds rules for 4 [network `tags`](https://cloud.google.com/vpc/docs/add-remove-network-tags) that can be
 applied to instances, similar to the division between subnetworks.
 
 * `public` - allow inbound traffic from all sources
 
+* `public-restricted` - allow inbound traffic from specific sources on the internet
```

Comment:
Make it clear that the "specific sources" are CIDR blocks

Reply:
Good point. Updated with a suggestion to name it `allowed_public_restricted_subnetworks`.